src/components/donations/grid/GridAppbar.tsx

```diff
@@ -28,6 +32,18 @@ export default function GridAppbar() {
       AlertStore.show(t('common:alerts.success'), 'success')
     },
   })
+  const [searchValue, setSearchValue] = useState('')
+
+  const debounceSearch = debounce((text: string) => {
+    donationStore.setDonationSearch(text)
+  }, 1500)
```

Comment:
I would make this 0.5 or 1 sec.

Reply:
Yea, actually I don't know why I left it at 1500ms. 500ms will be more than enough.